rust/sealevel/client/src/core.rs

```diff
@@ -26,6 +26,24 @@ pub(crate) fn adjust_gas_price_if_needed(chain_name: &str, ctx: &mut Context) {
             (PROCESS_DESIRED_PRIORITIZATION_FEE_LAMPORTS_PER_TX * 1_000_000)
         // Divide by the max compute units
         / DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT as u64;
+
+        if initial_instructions
```

Comment:
sucks that you ran into this, but also great that we got to battle test the per-chain fee logic, should make future deploys much less annoying